gridsync/gui/welcome.py

```diff
@@ -372,7 +372,7 @@ def verify_settings(self, settings, from_wormhole=True):
             settings, self.known_gateways, self, from_wormhole
         )
         self.setup_runner = SetupRunner(self.known_gateways, self.use_tor)
-        steps = self.setup_runner.calculate_total_steps(settings) + 2
+        steps = self.setup_runner.calculate_total_steps(settings) + 6
```

Comment:
Don't know where the "2" comes from .. maybe some constants here would help. The +4 is "starting", "downloading", "verifying", "success" approximately (the statuses from zkap).